nhais-adaptor/mailbox/NHAIS/HA01/outbox/edifact-for-GP05.txt

```diff
@@ -27,12 +27,6 @@ RFF+950:F4'
 S01+1'
 RFF+TN:21'
 NAD+GP+1231231,PLP348:900'
-UNT+9+00024986'

```

Comment:
Why does this need to change?

Reply:
So the transaction below should all be part of the message above and more accurate to what edifact will send us. I wrongly created 3 messages when I only wanted 2.